repos/core-setup.proj

```diff
@@ -4,8 +4,10 @@
   <PropertyGroup>
     <BuildArguments>-ConfigurationGroup=$(Configuration) -PortableBuild=$(PortableBuild) -SkipTests=true </BuildArguments>
     <BuildArguments Condition="$(Platform.Contains('arm'))">$(BuildArguments) -TargetArchitecture=$(Platform) -DisableCrossgen=true -CrossBuild=true</BuildArguments>
+    <BuildArguments Condition="'$(OS)' != 'Windows_NT' AND '$(Platform)' == 'x86'">$(BuildArguments) -TargetArchitecture=$(Platform) -DisableCrossgen=true -CrossBuild=true</BuildArguments>
```

Comment:
I'm not a fan of the duplication here... could the `-TargetArchitecture=$(Platform) -DisableCrossgen=true -CrossBuild=true` be moved to a property? (The conditions could alternatively be combined, but that would make a fairly long line which I'm not a fan of.)

Reply:
Here, I have duplicated the code because imagine that following:
x86 should enable CrossGen while for arm we have to set Crossgen.

This way we can have a more granular way to manage the BuildArguments for OS-ARCH

Reply:
I think that calls for extracting the parameters into variables, not duplication. And even then, can't it wait until differences actually exist so we know for sure it's arranged in a useful way?